docs/LearnCalico.md

```diff
@@ -0,0 +1,358 @@
+# Calico networking in a containerized environment
+
+This GitHub repository is the entry point for running a Calico network in a
+containerized environment.
+
+The `calico-docker` repository allows users to easily configure endpoints and 
```

Comment:
I'm not a fan of this paragraph. It tries to cover too much too fast. I tried rewording it a few times but I see this document as having a much better flow when this paragraph is left out.